web/ui/static/js/graph/index.js

```diff
@@ -286,7 +287,7 @@ Prometheus.Graph.prototype.initTypeahead = function(self) {
   self.expr.typeahead({
     autoSelect: false,
     source: source,
-    items: "all",
+    items: 100,
```

Comment:
Can we make this limit higher, like 1000? When exploring metrics, I often just type e.g. `node_` in the expression input to see what exists with that prefix. But on [http://demo.robustperception.io:9090/graph?g0.range_input=1h&g0.expr=count(count%20by(__name__)%20(%7B__name__%3D~%22node_.*%22%7D))&g0.tab=1](the RobustPerception demo server), this already returns 169 different metric names. It would be possible to use the dropdown selector for some of these use cases, but that is more cumbersome.